queries.sql

```diff
@@ -23,4 +23,25 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name NOT LIKE 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+-- second milestone
+
+-- How many animals are there?
+SELECT COUNT(*) from animals;
```

Comment:
- [ ] Among the project requirements, you are expected to write the queries which answer questions in TRANSACTIONS here in your `queries.sql`  so please add them too just as you did with the rest 😉 👍 
For example :

```sql
 BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals;
```

Reply:
Missing query applied :heavy_check_mark: 

![Screenshot from 2022-09-13 11-06-14](https://user-images.githubusercontent.com/37988027/189937818-1fec0755-48ec-4ea6-8566-26801ead3402.png)